indexing-service/src/main/java/org/apache/druid/indexing/worker/config/WorkerConfig.java

```diff
@@ -29,6 +29,7 @@
 
 /**
  */
+@SuppressWarnings("SS_SHOULD_BE_STATIC")
```

Comment:
In general, please

1) Make suppressions as local as possible: can suppress for individual fields?
2) Explain a suppression with a comment: mention what does the rule mean ("SS_SHOULD_BE_STATIC" is not self-explanatory) and explain why it should be suppressed in each particular case.